From 988f18e648f464c16589175c046897657b44ec9f Mon Sep 17 00:00:00 2001
From: "Pawse, GuruprasadX" <guruprasadx.pawse@intel.com>
Date: Wed, 11 Jan 2017 18:43:31 +0530
Subject: [PATCH 0242/1240] ASoC: Intel: Add Icelake machine id to use WM8281

Icelake supports WM8281 codec as well, hence updating
the machine id list.

Change-Id: I7f18b9cc11d06f2d3c535a611b5bd2894b9ab2ad
Signed-off-by: Pawse, GuruprasadX <guruprasadx.pawse@intel.com>
Reviewed-by: R, Dharageswari <dharageswari.r@intel.com>
Reviewed-by: Diwakar, Praveen <praveen.diwakar@intel.com>
Reviewed-by: Kp, Jeeja <jeeja.kp@intel.com>
Tested-by: Sm, Bhadur A <bhadur.a.sm@intel.com>
---
 sound/soc/intel/skylake/skl.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index 101312a..0031f35 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -1061,6 +1061,8 @@ static const struct sst_acpi_mach sst_cnl_devdata[] = {
 static struct sst_acpi_mach sst_icl_devdata[] = {
 #if IS_ENABLED(CONFIG_SND_SOC_RT700)
 	{ "dummy", "icl_rt700", "intel/dsp_fw_icl.bin", NULL, NULL, NULL },
+#elif IS_ENABLED(CONFIG_SND_SOC_WM5110)
+	{ "dummy", "icl_wm8281", "intel/dsp_fw_icl.bin", NULL, NULL, NULL },
 #else
 	{ "dummy", "icl_rt274", "intel/dsp_fw_icl.bin", NULL, NULL, NULL },
 #endif
-- 
2.7.4

